licenses-report.txt

```diff
@@ -50,210 +163,425 @@ The following software have components provided under the terms of this license:
+- sigar (from )
+- sjk-cli (from https://repo1.maven.org/maven2/org/gridkit/jvmtool/sjk-cli)
 - sjk-core (from https://repo1.maven.org/maven2/org/gridkit/jvmtool/sjk-core)
+- sjk-hflame (from https://repo1.maven.org/maven2/org/gridkit/jvmtool/sjk-hflame)
+- sjk-jfr-standalone (from https://repo1.maven.org/maven2/org/perfkit/sjk/parsers/sjk-jfr-standalone)
+- sjk-jfr5 (from https://repo1.maven.org/maven2/org/perfkit/sjk/parsers/sjk-jfr5)
+- sjk-jfr6 (from https://repo1.maven.org/maven2/org/perfkit/sjk/parsers/sjk-jfr6)
+- sjk-json (from https://repo1.maven.org/maven2/org/gridkit/jvmtool/sjk-json)
+- sjk-nps (from https://repo1.maven.org/maven2/org/perfkit/sjk/parsers/sjk-nps)
+- sjk-stacktrace (from https://repo1.maven.org/maven2/org/gridkit/jvmtool/sjk-stacktrace)
 - snappy-java (from https://github.com/xerial/snappy-java)
-- value (from )
+- stream-lib (from https://github.com/addthis/stream-lib)
+- swagger-annotations (from )
+- swagger-jaxrs (from )
+- swagger-jersey2-jaxrs (Jersey 2.x support) (from http://swagger.io)
 
 ========================================================================
 BSD-2-Clause
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- ANTLR 4 Runtime (from http://antlr.org)
 - ASM Analysis (from )
 - ASM Commons (from )
 - ASM Core (from )
 - ASM Tree (from )
 - ASM Util (from )
-- AntLR (from http://www.antlr.org/)
+- Apache Commons Math (from http://commons.apache.org/proper/commons-math/)
+- Hamcrest (from http://hamcrest.org/JavaHamcrest/)
+- Hamcrest Core (from http://hamcrest.org/)
 - HdrHistogram (from http://hdrhistogram.github.io/HdrHistogram/)
+- JLine (from )
 - jersey-container-servlet (from git://java.net/jersey~code/project/jersey-container-servlet)
+- jersey-container-servlet-core (from git://java.net/jersey~code/project/jersey-container-servlet-core)
+- jersey-core-client (from git://java.net/jersey~code/jersey-client)
+- jersey-core-server (from git://java.net/jersey~code/jersey-server)
+- jersey-ext-bean-validation (from )
+- jersey-ext-metainf-services (from )
+- jersey-inject-hk2 (from )
+- jersey-media-jaxb (from )
+- jersey-test-framework-core (from )
+- jersey-test-framework-provider-inmemory (from )
 
 ========================================================================
 BSD-3-Clause
 ========================================================================
 The following software have components provided under the terms of this license:
 
 - ANTLR 3 Runtime (from http://www.antlr.org)
+- ANTLR 3 Tool (from )
+- ANTLR 4 Runtime (from http://antlr.org)
 - ANTLR StringTemplate 4.0.2 (from http://www.stringtemplate.org)
 - ASM Analysis (from )
 - ASM Commons (from )
 - ASM Core (from )
 - ASM Tree (from )
 - ASM Util (from )
-- AntLR (from http://www.antlr.org/)
+- Apache Cassandra (from http://cassandra.apache.org)
```

Comment:
We have `Apache Cassandra` under the Apache 2.0 heading too. Is this ok?

Reply:
It probably has `Apache Cassandra` under Apache 2.0 since that's a direct dependency and it has it here to cover the transitive dependencies. As in C* depends on something that uses BSD-3.